components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -220,12 +214,13 @@ abstract class AbstractFetchDownloadService : Service() {
                                 context = context,
                                 filePath = currentDownloadJobState.state.filePath,
                                 contentType = currentDownloadJobState.state.contentType
-                            )) {
+                            )
+                        ) {
                             val fileExt = MimeTypeMap.getFileExtensionFromUrl(
-                                    currentDownloadJobState.state.filePath.toString()
+                                currentDownloadJobState.state.filePath.toString()
                             )
                             val errorMessage = applicationContext.getString(
-                                    R.string.mozac_feature_downloads_open_not_supported1, fileExt
```

Comment:
Please don't commit changes in indentation in lines that you don't intentionally modified as it makes the diff way harder to read. :)

Reply:
Sorry, my Android Studio somehow became incorrectly configured with indentations/spacing after the last update and I'm not sure why that happened or how I can reset it to what we need :(

Reply:
Don't worry, it looks like it was applying the styling to the whole file, it's better to just apply styling to the sections that you change, one way to verify it before pushing just review the changes and make sure that only the parts that you changed are getting committed, that make the review process easier :)